miio/integrations/vacuum/dreame/dreamevacuum.py

```diff
@@ -0,0 +1,589 @@
+"""Dreame Vacuum."""
```

Comment:
Please avoid renaming the file for now, as it makes it really hard to read the diffs.